main.go

```diff
@@ -110,11 +119,47 @@ func main() {
 	}
 
 	if !useTLS {
-		log.Info("serving gRPC-Gateway and OpenAPI Documentation on http://", gatewayAddr)
-		log.Fatalln(gwServer.ListenAndServe())
+		logger.Info("serving gRPC-Gateway and OpenAPI Documentation", zap.String("gatewayAddr", "http://"+gatewayAddr))
+		if err := gwServer.ListenAndServe(); err != nil {
+			logger.Fatal("failed to serve gRPC-Gateway", zap.Error(err), zap.Bool("tls", false))
+		}
 	}
 
 	gwServer.TLSConfig = tlsConfig
-	log.Info("serving gRPC-Gateway and OpenAPI Documentation on https://", gatewayAddr)
-	log.Fatalln(gwServer.ListenAndServeTLS("", ""))
+	logger.Info("serving gRPC-Gateway and OpenAPI Documentation", zap.String("gatewayAddr", "https://"+gatewayAddr))
+	if err := gwServer.ListenAndServeTLS("", ""); err != nil {
+		logger.Fatal("failed to serve gRPC-Gateway", zap.Error(err), zap.Bool("tls", true))
+	}
+}
+
+func initLogger() *zap.Logger {
+	logConf := zap.NewProductionEncoderConfig()
+	logConf.EncodeTime = func(time time.Time, encoder zapcore.PrimitiveArrayEncoder) {
+		encoder.AppendInt64(time.Unix())
+	}
+
+	levelEnablerFunc := zap.LevelEnablerFunc(func(_ zapcore.Level) bool {
+		return true
+	})
+
+	logEncoder := zapcore.NewConsoleEncoder(logConf)
+
+	if os.Getenv("LOGGER_MODE") == "PRODUCTION" {
```

Comment:
@browseman please pass that for JSON logging

Reply:
You want this to be set as env. variable where, dev, prod, or both?

Reply:
That's for better integration with PaperTrail, so both

Reply:
ok, will change that in the next request for deploy